routes/api.php

```diff
@@ -36,6 +36,8 @@
     Route::middleware(['auth:api', 'verified'])->group(function () {
         Route::post('subscribe', 'UserController@subscribe');
         Route::get('users/me', 'UserController@me');
+        Route::get('users/list-notifications', 'UserController@listNotifications');
+        Route::post('users/read-notification', 'UserController@readNotification');
```

Comment:
Would it be better to set api end points like this:

Route::get('users/notifications', 'UserController@listNotifications'); -> this would list the unread notifications
Route::post('users/notification/{notification}/read', 'UserController@readNotification'); -> would mark the notification as 'read'.

You can also use 'model' name in the controller handler to load the object directly for the notification.
see: https://github.com/paypal/api-standards/blob/master/api-style-guide.md#uri-naming-conventions